### DISCUSSION TITLE
Corsi e sezioni classi

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#210] [Idee e proposte]
Salve, sarebbe possibile introdurre la possibilità di inserire le stesse sezioni (A,B,C..) a classi diverse cambiando semplicemente il corso di indirizzo? Ad esempio nella mia scuola abbiamo classi che dovrebbero essere inserite nel registro elettronico come 1AINF o 1AGRAF, però Il registro Giua non ci permette di creale perché hanno la stessa sezione (in questo caso la A) inserita. Non so se mi sono spiegato bene. Attendo risposta

**Comment by trinko** [comment]
In futuro penso di si.

**Reply by scuolagioco** [reply]
Spero già dal prossimo aggiornamento!
